Library/Formula/tmux.rb

```diff
@@ -22,6 +22,10 @@ class Tmux < Formula
   depends_on "pkg-config" => :build
   depends_on "libevent"
 
+  # This fixes the Tmux 2.1 update that broke the ability to use select-pane [-LDUR]
+  # to switch panes when in a maximized pane
```

Comment:
Please add the link to the patch in the comment: https://github.com/tmux/tmux/issues/150#issuecomment-149466158